improver/metadata/probabilistic.py

```diff
@@ -206,6 +214,24 @@ def probability_is_above_or_below(cube: Cube) -> Optional[str]:
     return threshold_attribute
 
 
+def is_percentile(cube: Cube) -> bool:
+    """Determines whether a cube contains probability data at a range of
+    percentiles.
+
+    Args:
+        cube:
+            Cube to check for probability percentile data.
+
+    Returns:
+        True if in threshold representation.
+    """
+    try:
+        find_percentile_coordinate(cube)
+    except (CoordinateNotFoundError, ValueError):
```

Comment:
I've had a look back through repository history for the `find_percentile_coordinate` and `find_threshold_coordinate` functions. It looks like they were originally written in different times/locations, then later bunched together in improver/metadata/probabilistic.py during some refactoring. It would be worthwhile making these two functions (plus the related `is_percentile`/`is_probability`) consistent with each other, but I think that's outside the scope of this PR. I'll raise an issue for this to be addressed separately.

As long as the plugin/CLI does something sensible (either processes on one of the coordinates or raises an exception), I think this is OK. I'm not even sure what the data would mean if it had two percentile coordinates.

Reply:
Issue for this is #1723.